airbyte-integrations/bases/base-java/src/main/resources/log4j2.xml

```diff
@@ -3,7 +3,7 @@
 
   <Appenders>
     <Console name="ConsoleJSONAppender" target="SYSTEM_OUT">
-      <JsonTemplateLayout eventTemplateUri="classpath:AirbyteLogMessageTemplate.json"/>
+      <JsonTemplateLayout eventTemplateUri="classpath:AirbyteLogMessageTemplate.json" maxStringLength="3750"/>/>
```

Comment:
Can we leave a comment here or in the code explaining the significance of this number?

Reply:
Added